prepare-build.sh

```diff
@@ -148,25 +179,8 @@ PACKAGE_CLASSES = "package_ipk"' >> build/conf/local.conf
 SRCDIR = "${@os.path.abspath(os.path.join("${TOPDIR}", "../src/"))}"
 
 BBLAYERS = " \
-  ${SRCDIR}/meta-qt5 \
-  ${SRCDIR}/oe-core/meta \
-  ${SRCDIR}/meta-asteroid \
-  ${SRCDIR}/meta-asteroid-community \
-  ${SRCDIR}/meta-openembedded/meta-oe \
-  ${SRCDIR}/meta-openembedded/meta-multimedia \
-  ${SRCDIR}/meta-openembedded/meta-gnome \
-  ${SRCDIR}/meta-openembedded/meta-networking \
-  ${SRCDIR}/meta-smartphone/meta-android \
-  ${SRCDIR}/meta-openembedded/meta-python \
-  ${SRCDIR}/meta-openembedded/meta-filesystems \' > build/conf/bblayers.conf
-
-        # Find all layers under src/meta-smartwatch, remove the src/ prefix, sort alphabetically, and store it in an array.
-        IFS=$'\n' layers=($(find src/meta-smartwatch -mindepth 1 -name "*meta-*" -type d | sed -e 's|src/||' | sort))
-        for layer in ${layers[*]}; do
-            echo "  \${SRCDIR}/$layer \\" >> build/conf/bblayers.conf
-        done
-
-        echo "\"" >> build/conf/bblayers.conf
+"' > build/conf/bblayers.conf
```

Comment:
This is a bit confusing to read.  An alternative approach that might be easier would be to pull the list of layers into an array with something like:
```
existing_layers=($(sed -nr 's/  \$\{SRCDIR\}\/(\S+) \\/\1/p' build/conf/bblayers.conf))
```
Then `update_layer_config` would only have to compare the two arrays to figure out what to add.